Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -430,7 +507,9 @@ open class MapView: UIView {
 
         if window != nil {
             validateDisplayLink()
+            subscribeToLifecycleNotifications()
         } else {
+            unsubscribeFromLifecycleNotifications()
             // TODO: Fix this up correctly.
```

Comment:
You're going to have conflicts with your other PR. Do you have a preference which one to merge first?

Reply:
This one for sure, the other PR is smaller and should be easier to resolve conflicts in.